core/src/main/scala/org/apache/spark/sql/internal/SnappySessionCatalog.scala

```diff
@@ -415,6 +415,15 @@ class SnappySessionCatalog(val externalCatalog: SnappyExternalCatalog,
     } else super.listDatabases()
   }
 
+  override def listTables(schema: String, pattern: String): Seq[TableIdentifier] = {
+    // convert to lower-case (for temp tables since hive catalog entries are already lower-case)
+    super.listTables(schema, pattern).map {
```

Comment:
Yes there are unit tests in spark-compatibility-tests. This has been removed in the "spark-lowercase-compat" changes though because entries are all lower-case.